provisioning/resources/configs/compositions/docker-compose-gcp.yml

```diff
@@ -138,7 +138,7 @@ services:
     logging:
       driver: gcplogs
     environment:
-      - "SP_JAVA_OPTS=-Xmx${SP_JVM_SIZE}"
+      - "JAVA_OPTS=-Xmx${SP_JVM_SIZE} -Dlog4j2.formatMsgNoLookups=true"
```

Comment:
This will have an interesting side-effect of actually setting the correct heap size for these apps as-well - hopefully not a bad thing!